pynuodb/session.py

```diff
@@ -194,17 +194,13 @@ def send(self, message):
         if self.__cipherOut:
             message = self.__cipherOut.transform(message)
 
+        if self.__pyversion == '3':
```

Comment:
Moving this up is to fix what appears to be a potential bug where we serialize the length of the string into the message, rather than the number of bytes to encode that string.

Reply:
Interesting.  I was trying to test Java stored procedures and when I tried to load it using pynuodb I got some weird error about invalid format.  I wonder if it's related.